app/templates/events/view/speakers.hbs

```diff
@@ -16,19 +16,12 @@
         {{/link-to}}
       {{/tabbed-navigation}}
     </div>
-    <div class="eight wide column">
-      <button class="ui blue button right floated">{{t 'Add Speaker'}}</button>
-      <button class="ui blue button right floated">{{t 'Export as CSV'}}</button>
-      {{#ui-dropdown class='labeled icon button right floated'}}
-        <i class="dropdown icon"></i>
-        <div class="default text">{{t 'No. of entries'}}</div>
-        <div class="menu">
-          <div class="item">10</div>
-          <div class="item">25</div>
-          <div class="item">50</div>
-          <div class="item">100</div>
-        </div>
-      {{/ui-dropdown}}
+    <div class="ten wide {{if device.isMobile 'center aligned'}} column">
+      <button class="ui blue button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Export as CSV'}}</button>
+      {{#if device.isMobile}}
+        <div class="ui fitted divider"></div>
```

Comment:
But that is not causing any issue. It's not making any difference even on adding hidden 
![image](https://user-images.githubusercontent.com/12701036/28756169-a977e8b6-7586-11e7-8d4f-76e0b5c0e86a.png)

![image](https://user-images.githubusercontent.com/12701036/28756163-98043dc8-7586-11e7-92c4-6b4c0bc70184.png)